spec/PythonSpec.hs

```diff
@@ -0,0 +1,115 @@
+{-# LANGUAGE QuasiQuotes, OverloadedStrings #-}
+
+module PythonSpec (spec) where
+
+import           Test.Hspec
+import           Language.Mulang
+import           Language.Mulang.Parsers.Python
+
+import           Data.Text (Text, unpack)
+import           NeatInterpolation (text)
+
+run :: Text -> Expression
+run = py . unpack
+
+
+spec :: Spec
+spec = do
+  describe "parse" $ do
+    it "parses numbers" $ do
+      py "1" `shouldBe` MuNumber 1
+
+    it "parses booleans" $ do
+      py "True" `shouldBe` MuBool True
+
+    it "parses strings" $ do
+      py "\"some string\"" `shouldBe` MuString "\"some string\""
```

Comment:
Why the extra quotes?

Reply:
The parsed strings include them.
`"something"` gets parsed to `Strings ["\"something\""]`.

I didn't remove those because the parser makes no distinction between normal strings and multline strings, and in case someone actually wanted a string which starts with quotes. E.g. `"\"something\""`.

Reply:
Rethinking that though, if one were to want to parse a string which starts with quotes, this would be parsed as a triple escaped quote.
`"\"something\""` would become `Strings ["\"\\\"something\\\"\""]`.

So I guess I _could_ take them out?

Thoughts?

Reply:
Also, I thought multiline strings got parsed to a list with as many elements as there were lines in the string (that's the why of the intercalate "\n" [here](https://github.com/mumuki/mulang/pull/161/files#diff-8dd23b6f4626a0dafdcfc7669773b263R150)), but apparently that's not the case. They also get parsed to a single element list. 
So I have no clue as to why they get parsed to a list in the first place.